presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -171,7 +171,7 @@ class TaskManager {
   // coordinator for a considerable time.
   void cancelAbandonedTasks();
 
-  std::unique_ptr<protocol::TaskInfo> createOrUpdateTask(
+  std::unique_ptr<protocol::TaskInfo> createOrUpdateTaskImpl(
```

Comment:
Why rename to createOrUpdateTaskImpl? Shall we rename findOrCreateTask as well? Thanks!

Reply:
Because we had two createOrUpdateTask() and w=one of them is internal.